src/vivarium_nih_us_cvd/components/treatment.py

```diff
@@ -331,13 +340,20 @@ def on_initialize_simulants(self, pop_data: SimulantData) -> None:
             sbp_pipeline=self.gbd_sbp,
         )
 
-        # Enroll in outreach intervention
-        maybe_enroll_in_outreach = maybe_enroll_in_outreach_sbp.union(
-            maybe_enroll_in_outreach_ldlc
-        )
-        pop.loc[mask_emergency] = self.enroll_in_outreach(
-            pop_visitors=pop.loc[mask_emergency], maybe_enroll=maybe_enroll_in_outreach
-        )
+        # Enroll in interventions. The sbp treatment ramp includes both
+        # outreach and polypill enrollment logic whilc the ldlc ramp
+        # only includes outreach
+        if self.scenario.is_outreach_scenario:
```

Comment:
Why would we be enrolling anyone in an intervention during initialization? Coverage should always be 0